example/package.json

```diff
@@ -57,7 +53,7 @@
     "yup": "^0.29.3"
   },
   "devDependencies": {
-    "@availity/workflow": "^8.4.0",
+    "@availity/workflow": "link:../packages/workflow/",
```

Comment:
this should go back to the real version

Reply:
So my goal is for the `example` package to always run against the latest `@availity/workflow` code so that if we make a change to workflow that breaks something like `yarn build:production` in a web app, we know immediately and fail the merge/release until we figure out a fix

I think since `example` is a private package that won't be published that a symlink won't matter much here, but I'm not sure if there's a better solution